src/styles/components/_page-header.scss

```diff
@@ -41,3 +41,17 @@
   margin: 0 auto 12px;
   max-width: 600px; // fits a top line of text on a large screen to prevent FOUC
 }
+
+body:not(.unresolved) .w-unsupported-notice {
+  display: none !important;
+}
+
+@keyframes unsupported-appear {
+  from { visibility: hidden; }
+  to   { visibility: hidden; }
+}
+
+.w-unsupported-notice {
```

Comment:
Is it worth fading this out or should it just be persistent?

Reply:
I think it's fine. It only shows up after 5 seconds anyway and we don't expect many users to see it.